paper/paper.md

```diff
@@ -0,0 +1,323 @@
+integrate the skills together in their own work.
+The new curriculum we developed (https://umcarpentries.org/intro-curriculum-r/), which is hosted on GitHub,
+aims to teach novices how to implement reproducible research practices to
+perform data analysis.
+It emphasizes the iterative nature of coding and data analysis through
+integrated live coding lessons and practice exercises.
+Furthermore, this curriculum provides a succinct resource for learners to reference both during and after
+the workshop, and lowers the barrier to entry for new instructors as they do not
+have to develop their own curriculum or sift through extensive content.
+We developed this curriculum during a two-day sprint,
+successfully used the curriculum to host a two-day virtual workshop,
+and updated the material based on instructor and learner feedback.
+We hope that this curriculum will be useful to other instructors interested in
+teaching workshops with similar learning objectives.
+
+# Statement of Need
+
+The University of Michigan instance of The Carpentries has taught workshops
+using the curriculum created by The Carpentries organization since 2015.
+In that time, our instructors have found several advantages and disadvantages to
+using the Software Carpentry curriculum in particular.
+Some advantages are that any programming language lesson (e.g. R or
+Python) can be paired with lessons on the Unix shell and version control,
+the lessons have been refined over years by many contributors and taught at
+workshops around the world, and
+the instructional design demonstrates good pedagogy for teaching beginner data
+science practitioners.
+
+However, we found that the curriculum did not meet all of our needs.
+First, the modular nature of the curriculum means that each lesson is independent
+from the others, so it is not apparent to learners how to integrate all of the
+skills together for reproducible research projects.
+Additionally, The Carpentries material is more of a reference than a lesson plan as
+there is too much content for the time allotted in a two-day workshop.
+Therefore, workshops contain inconsistent content depending on who teaches, and new
+instructors face an overwhelming amount of work to prepare for their first
+workshop.
+Finally, some lessons include niche content that we do not feel is necessary to teach
+beginners as we do not use it ourselves in our research projects.
+
+We sought to create a new curriculum that would
+teach computational skills in an integrated manner to demonstrate the
+reproducible research workflows we use in our own work,
+utilize the same inclusive pedagogy refined by The Carpentries,
+and contain an appropriate amount of content to deliver consistent workshops
+and reduce the burden for new instructors to get involved.
+
+# Collaborative Curriculum Development
+
+We drew on the expertise of The Carpentries community at the
+University of Michigan to develop a custom curriculum that would meet the needs
```

Comment:
Do we have data about what their actual needs are, or are these based on our perceived ideas of what their needs *should* be? Perhaps there was feedback from previous events that could support this claim.